### ISSUE TITLE
ImageFileDirectory_v2.load can get OverflowError on 32 bit systems with bad exif

### ISSUE THREAD

**Issue by paddywwoof** [python-pillow/Pillow#3994]
Because offset is unpacked from four bytes and then used as ssize_t in BytesIO.seek() it can overflow on 32 bit systems. This means that images that appear quite viewable with other software or on 64 bit systems using PIL will not render on Raspberry Pi, or other SOC being used for digital pictureframes etc.

offset should not be greater than ``(1 << ctypes.sizeof(ctypes.c_ssize_t) - 1) - 1``

Here is an example demonstrating the problem https://www.eldwick.org.uk/pix/PICS0027.JPG also attached
![PICS0027](https://user-images.githubusercontent.com/1939398/62138683-f20e8600-b2df-11e9-95cd-42e9c79fc72e.JPG)

The simplest solution would be to put a try/except
```
                    try:
                        fp.seek(offset)
                    except OverflowError as e:
                      if DEBUG:
                          print("- offset value wrong. Possibly corrupt EXIF data. ", e)
                      continue  # ignore corrupt EXIF data
                    data = ImageFile._safe_read(fp, size)
                    fp.seek(here)
```
Happy to submit a pull request along these lines.

**Comment by radarhere** [comment]
Just to link the issues, PR #3995 has been created

**Comment by radarhere** [comment]
In a discussion in the PR, it is found that this image is actually fixed in Pillow 6.0 by #3584
